tests/cockpit-tasks-centosci.yaml

```diff
@@ -0,0 +1,83 @@
+---
+apiVersion: v1
+kind: List
+items:
+- kind: ReplicationController
+  metadata:
+    name: cockpit-tasks
+  spec:
+    replicas: 1
+    selector:
+      infra: cockpit-tasks
+    template:
+      metadata:
+        labels:
+          infra: cockpit-tasks
+      spec:
+        containers:
+        - name: cockpit-tasks
+          image: docker.io/cockpit/tests
+          securityContext:
+            runAsUser: 1111
+            hostNetwork: false
```

Comment:
Please drop this on CentOS CI, it doesn't work there.

In fact, I think we should also work towards fixing the UID on our BOS cloud. The necessary pieces should all be there now, and there's also PR #176. But this is clearly a follow-up.